src/pages/Home/index.js

```diff
@@ -169,10 +169,10 @@ const Home = () => {
         <div className="col-lg-3 pr-lg-2">
           <DataTile
             icon={isLightTheme ? ActiveSmeshersIcon : ActiveSmeshersIconWhite}
-            title="Active Smeshers"
+            title="Activation transactions"
             url={`${deployConfig.explorerUrl}smeshers`}
             value={activeSmeshers && activeSmeshers.amt}
-            toolTipMess="Total of active smeshers in the most recent epoch. The graph displays the number of active smeshers in previous epochs. Active smeshers are deployed Spacemesh full p2p nodes that participate in the Spacemesh consensus protocol and submit blocks with transactions to the network."
+            toolTipMess="Total number of PoST activation transactions in the most recent epochs. PoST activation transactions are nodes that participate in the Spacemesh consensus protocol and submit blocks with transactions to the network."
```

Comment:
epoch or epochs?

Reply:
I'd insist on Activation Transaction without `PoST` here.

"PoST activation transactions are nodes " this is definitely not correct though :)

Reply:
> epoch or epochs?

??